Formula/tbb.rb

```diff
@@ -42,13 +42,15 @@ def install
     cd "python" do
       ENV.append_path "CMAKE_PREFIX_PATH", prefix.to_s
       ENV["LDFLAGS"] = "-rpath #{opt_lib}" if OS.mac?
```

Comment:
At some point I'd like to change this to a relative path using `@loader_path` to avoid the overhead of RPATH relocation at pour-time.